initial/src/main/java/com/example/resources/CalculatorResource.java

```diff
@@ -0,0 +1,40 @@
+package com.example.resources;
+
+import com.example.services.CalculatorService;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Random;
+import java.util.List;
+
+@RestController
+public class CalculatorResource {
+    public static final String URL_PREFIX="/calculator";
+
+	@RequestMapping(URL_PREFIX + "/random")
```

Comment:
Not a blocker. But going forward, can you add *javadoc* to these public methods?

Reply:
added!